cspell.json

```
@@ -0,0 +1,62 @@
+{
+    "version": "0.1",
+    "language": "en",
+    "enabledLanguageIds": [
+        "css",
+        "graphql",
+        "html",
+        "json",
+        "javascript",
+        "javascriptreact",
+        "markdown",
+        "mdx",
+        "plaintext",
+        "restructuredtext",
+        "sass",
+        "scss",
+        "typescript",
+        "typescriptreact",
+        "yaml",
+        "yml"
+    ],
+    "ignorePaths": [
+        "cdai",
+        "coverage",
+        "css",
+        "es",
+        "lib",
+        "node_modules", // this will ignore anything the node_modules directory
```

There isn't a comment syntax for JSON. Also, this is how all the ignorePaths work.

```suggestion
        "node_modules",
```